charts/kube-plex/README.md

```diff
@@ -29,12 +35,18 @@ The following tables lists the configurable parameters of the Plex chart and the
 | `ingress.tls`                  | Ingress TLS configuration | `[]` |
 | `rbac.create`                  | Create RBAC roles? | `true` |
 | `nodeSelector`             | Node labels for pod assignment | `beta.kubernetes.io/arch: amd64` |
+| `configMap.plexPreferences.enabled`             | Enable init script(and load configMap) that will read all environment variables starting with PLEX_PREFERENCE_ | `false` |
+| `configMap.plexPreferences.name`             | Name of configMap | `41-plex-preferences` |
+| `configMap.plexPreferences.defaultMode`             | Unix permissions in container (0755) | `493` |
+| `configMap.plexPreferences.mountPath`             | The full path to the file we will mount the configMap to | `/etc/cont-init.d/41-plex-preferences` |
```

Comment:
When would someone want to customise these? Would it make more sense for us to set these appropriately always when the chart is installed? I'm conscious of exposing too many options and causing confusion, when instead the Helm chart should take care of non-relevant (to the user) bits like the mountPath, file mode etc.